src/item.h

```diff
@@ -670,6 +677,45 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
      */
     int chip_resistance( bool worst = false ) const;
 
+    /** How much damage has the item sustained? */
+    int damage() const {
+        return damage_;
+    }
+
+    /** Minimum amount of damage to an item (state of maximum repair) */
+    int min_damage() const;
+
+    /** Maximum amount of damage to an item (state before destroyed) */
+    int max_damage() const;
+
+    /**
+     * Apply damage to item constrained by @ref min_damage and @ref max_damage
+     * @param qty maximum amount by which to adjust damage (negative permissible)
+     * @param dmg type of damage which may be passed to @ref on_damage callback
+     * @return whether item should be destroyed
+     */
+    bool mod_damage( int qty, damage_type dt = DT_NULL );
```

Comment:
Does it really need to return a value? This obscures logic and enforces checking comments in the header:
```c++
if( it.mod_damage( ... ) ) { // Does true here mean that the damage was successfully modified?
```
This version is slightly longer, but leaves no questions:
```c++
it.mod_damage( ... ); // returns void
if( it.destroyed() ) { // pure getter
```
This also has other advantages: 
- we can always check whether the item was destroyed and remove the object in one known place (probably along with an appropriate message: "Your item was destroyed.").
- other causes of destroy can be easily introduced and checked at once.